client/modules/wj-css/src/themes/_colors.scss

```diff
@@ -0,0 +1,79 @@
+// Global Colors
+// prettier-ignore
+:root {
+  // Monochrome
+  --col-white:     rgb(245, 245, 245);
+  --col-lightgray: rgb(185, 185, 185);
+  --col-gray:      rgb(60, 50, 55);
+  --col-black:     rgb(20, 20, 20);
+
+  // Accent
+  --col-accent-1: rgb(25, 169, 255);
+  --col-accent-2: rgb(0, 104, 181);
+
+  // Contextual
+  --col-hint:    rgb(50, 150, 255);
+  --col-success: rgb(30, 180, 65);
+  --col-danger:  rgb(180, 45, 10);
+  --col-warning: rgb(240, 125, 0);
+  --col-info:    rgb(20, 125, 205);
+
+  // Misc
+  --col-text-highlight: rgb(255, 255, 0);
+  --col-text-select:    rgba(50, 150, 255, 0.25);
+  --col-link-ext:       rgb(50, 150, 255);
+  --col-link-int:       rgb(180, 0, 20); // TODO: find replacement color
```

Comment:
Can you elaborate here? What is inadequate about this color that needs replacing?

Reply:
Ah yeah sorry forgot about this color. It's a certain shade of SCP wiki red rather than some WJ relevant color.

Reply:
Sounds good. Could you humor me and modify the comment to mention this?

Reply:
Was planning to but I'm on my phone rn. I might just pick a color (I also realized I need to move the font files to this SCSS package).